seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2636,7 +2636,7 @@
         "<blockquote>function plusThree(num) {<br>  return num + 3;<br>}<br>var answer = plusThree(5); // 8</blockquote>",
         "<code>plusThree</code> takes an <dfn>argument</dfn> for <code>num</code> and returns a value equal to <code>num + 3</code>.",
         "<hr>",
-        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value."
+        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value. You can call the <code>timesFive</code> function to test it."
```

Comment:
@systimotic Reading the original comment of this thread, I thought of a better phrasing:

> See the the last line in the editor for an example of how you can test your `timesFive` function

@erictleung @systimotic I'd suggest we update this sentence but otherwise keep the seed as it is. Would that be enough to get this merged? :blush: